src/entity/posts/utils.ts

```diff
@@ -257,11 +257,21 @@ const addPostAndKeywordsToDb = async (
     visibleAt: new Date(),
   });
   await entityManager.save(post);
-  if (data.tags?.length) {
+  await addTagsAndKeywords(entityManager, data?.tags, mergedKeywords, data.id);
+  return data.id;
+};
+
+export const addTagsAndKeywords = async (
+  entityManager: EntityManager,
+  tags: string[],
+  mergedKeywords: string[],
+  postId: string,
+): Promise<void> => {
+  if (tags?.length) {
     await entityManager.getRepository(PostTag).insert(
```

Comment:
By now this is no longer used and is actually deprecated. We use only the keywords

Reply:
Will remove, also will impact the existing new post then.